StortSpelProjekt/Game/src/Gamefiles/Components/PlayerInputComponent.cpp

```diff
@@ -8,6 +8,8 @@
 #include "Physics/Physics.h"
 #include "../Misc/Option.h"
 
+#include "stdafx.h"
+
```

Comment:
Feels a bit weird that you'd use the pch in Game project. But after some reading on the matter I can accept it. Make sure that this is the first include though!